api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/freemarker/DeletePropertyController.java

```diff
@@ -64,8 +69,8 @@ private String getRedirectUrl(VitroRequest vreq) {
 		// TODO Auto-generated method stub
     	String subjectUri = EditConfigurationUtils.getSubjectUri(vreq);
     	String predicateUri = EditConfigurationUtils.getPredicateUri(vreq);
-    	int hashIndex = predicateUri.lastIndexOf("#");
-    	String localName = predicateUri.substring(hashIndex + 1);
+		Property prop = ResourceFactory.createProperty(predicateUri);
```

Comment:
Nice job, @matthiasluehr .

If you could fix the formatting to align with the lines before and after, we can get this merged.